docs/apache-hive-catalog.md

```diff
@@ -140,25 +140,31 @@ Since 0.6.0-incubating, the data types other than listed above are mapped to Gra
-| `numFiles`              | Used to store the number of files in the table. | 0.2.0         |
-| `totalSize`             | Used to store the total size of the table.      | 0.2.0         |
-| `EXTERNAL`              | Indicates whether the table is external.        | 0.2.0         |
-| `transient_lastDdlTime` | Used to store the last DDL time of the table.   | 0.2.0         |
+| Property Name           | Description                                                                                                                                | Default Value                                                                                                                                       | Required | Reserved | Immutable | Since Version |
+|-------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|----------|----------|-----------|---------------|
+| `comment`               | Used to store a table comment.                                                                                                             | --                                                                                                                                                  | --       | Yes      | --        | 0.2.0         |
+| `numFiles`              | Used to store the number of files in the table.                                                                                            | --                                                                                                                                                  | --       | Yes      | --        | 0.2.0         |
+| `totalSize`             | Used to store the total size of the table.                                                                                                 | --                                                                                                                                                  | --       | Yes      | --        | 0.2.0         |
+| `EXTERNAL`              | Indicates whether the table is external.                                                                                                   | --                                                                                                                                                  | --       | Yes      | --        | 0.2.0         |
+| `transient_lastDdlTime` | Used to store the last DDL time of the table.                                                                                              | --                                                                                                                                                  | --       | Yes      | --        | 0.2.0         |
```

Comment:
Suggest putting these `properties` behind the table because they are reserved.

Reply:
updated